swap_meet/item.py

```diff
@@ -1,2 +1,24 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id=None, condition=0):
+        self.id = uuid.uuid1().int if id is None else id
+        self.condition = condition
+    
+    def get_category(self):
+        return "Item"
+
+    def __str__(self): 
+        return f"An object of type Item with id {self.id}."
```

Comment:
Overriding the supplied `__str__` method, with your own! We could make this even more dynamic by replacing `Item` with `self.get_category()`. We could then use this method in tandem with our children classes `__str__` method. It would look like this:
```suggestion
    def __str__(self): 
        return f"An object of type {self.get_category} with id {self.id}."
```
Check my comment in `Clothing` to see how it would be used in a child class.

Reply:
That makes sense. Refactored our code.